k8s/charts/rtsp-camera-simulator/templates/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rtsp-camera-simulator
```

Comment:
You should add the release name to all names so that a user can run multiple releases.

Reply:
I just give a fixed name so as to prevent user to create multiple instances in one namespace

Reply:
But we want to allow users to create multiple instances. Perhaps they want two instances with different default height and width.